lib/supabase/artist/updateArtistProfile.tsx

```diff
@@ -52,14 +52,14 @@ const updateArtistProfile = async (
         infoUpdate.knowledges = account_info.knowledges;
       }
       
-      infoUpdate.label = label || account_info.label;
+      infoUpdate.label = label === "" ? null : label;
       await updateAccountInfo(artistId, infoUpdate);
     } else {
       await insertAccountInfo({
         image,
         instruction,
         knowledges,
-        label,
+        label: label === "" ? null : label,
```

Comment:
DRY principle
- actual: same code block repeated twice (line 55 and 62)
- required: create a shared variable so maintenance of code is easier.